tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -151,6 +151,11 @@
     <!-- Latest package https://www.nuget.org/packages/NUnit3TestAdapter/4.3.1 -->
     <PackageReference Include="NUnit3TestAdapter" Version="4.3.1" />
 
+    <!-- Integration: OpenTelemetry -->
```

Comment:
May need to be reworked as done in #3604

Reply:
Rebased and fixed 🙂